src/smee.ts

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env node
+
+import Client from './client'
+import { parseCommandLine } from './cmd'
+
+async function runClient ({ source, target }: { source: string, target: string }): Promise<void> {
+  if (typeof source !== 'string' || source.length === 0) {
+    source = await Client.createChannel()
+  }
```

Comment:
As far as I can tell, they have to see the console output for the resulting execution.

I wasn't going to change this behavior at this time since we don't rely on it in any way.

Reply:
Admittedly I didn't try very hard, but I didn't see where in the code it actually reported what the URL was.

This doesn't seem like a very useful feature (i.e., you get a new URL every time, which means you have to go reconfigure the originating source), but I totally understand the lack of need to change it now.